malibal/aon/s1/README.wiki

```diff
@@ -0,0 +1,44 @@
+= Malibal Aon S1 =
+
+== Tested Hardware ==
+
+* CPU: Intel(R) Core(TM) i7-13700H CPU @ 2.4GHz
+* RAM: 32 GB
+* HDD: 2 TiB SSD
+* Screen: 14" WQXGA+ (2880x1800)
+* Graphics: Intel Raptor Lake-P [Iris Xe Graphics]. nVidia graphics disabled preemptively.
+* Display: Wayland.
+* Input: trackpad
+
+== Firmware Configuration ==
+
+No tweaking to get a useful system, but some is needed.
+- Sleeping doesn't work properly and overheats the system.
+- Fans spin even when the system is at almost no load and CPUs report 30°C-35°C
+
+== Installation ==
+
+=== Before installation ===
+
+I don't recall what I did very well, probably just disabled secure boot as
+there's no nice and meaningful way of using it yet.
+
+=== After installation ===
+
+=== Optional ===
+
+* Run fwupd
+
+== Troubleshooting ==
+
+
+== Broken Functionality ==
+
+=== Sleeping works, but overheats the machine ===
```

Comment:
Maybe some hardware prevents deeper sleep states?

Reply:
I'm not sure if it's just prevention, or there's something really wrong because I tested and leaving the machine idle, but on produces less heat than making it sleep.

I suspect there's something broken with the BIOS firmware right now.